dll/cpl/sysdm/environment.c

```diff
@@ -103,17 +104,18 @@ GatherDataFromListBox(HWND hwndDlg,
         GlobalFree(VarData->lpRawValue);
         VarData->lpRawValue = GlobalAlloc(GPTR, (dwValueLength + 1) * sizeof(TCHAR));
     }
-                    
+
+    /* Copy the first value */
     SendDlgItemMessage(hwndDlg, IDC_LIST_VARIABLE_VALUE, LB_GETTEXT, 0, (LPARAM)szData);
     StringCchCopy(VarData->lpRawValue, dwValueLength + 1, szData);
```

Comment:
Considering this is still 100% the same as the other lines, a more common pattern is to move this into the for loop, and check for `i > 0` before appending the `;`

Reply:
Alright